### ISSUE TITLE
Fancoil cold/hot water control failed warnings

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5853]
## Issue overview

While testing DOA to ZoneHVAC for #5676 I ran the existing DOAToFanCoilSupply and DOAToFanCoilInlet test files as annual simulations.  They both generate errors like this

```
   ** Warning ** Cold Water control failed in fan coil unit SPACE4-1 FAN COIL
   **   ~~~   **   Bad cold water mass flow limits
   **   ~~~   ** ..Water flow rate set to lower limit  Environment=Chicago Ohare Intl Ap IL USA TMY3 WMO#=725300, at Simulation time=03/19 15:45 - 16:00
   ** Warning ** Hot Water control failed in fan coil unit SPACE2-1 FAN COIL
   **   ~~~   **   Bad hot water mass flow limits
   **   ~~~   ** ..Water flow rate set to lower limit  Environment=Chicago Ohare Intl Ap IL USA TMY3 WMO#=725300, at Simulation time=04/15 07:00 - 07:04
```
### Details

Some additional details for this issue (if relevant):
- Platform (Win64)
- Version of EnergyPlus (v8.6.0 5b5301a77f )
- These errors still show after the changes in #5800.
### Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [ ] Defect file added (list location of defect file here)
- [ ] Ticket added to Pivotal for defect (development team task)
- [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by rraustad** [comment]
Why these new warnings? Seems related to chilled water temp and not zone load control.

```
 ** Severe  ** FindRootSimpleController: Controller error for controller = "DOAS COOLING COIL CONTROLLER"
 **   ~~~   **  Environment=Chicago Ohare Intl Ap IL USA TMY3 WMO#=725300, at Simulation time=01/17 12:45 - 13:00
 **   ~~~   **   Controller function is inconsistent with user specified controller action = Reverse action
 **   ~~~   **   Actuator will be set to maximum action
 **   ~~~   ** Controller control type=Temperature
 **   ~~~   ** Controller temperature setpoint = 11.60 [C]
 **   ~~~   ** Controller sensed temperature = 12.15 [C]
 **   ~~~   ** Controller actuator mass flow rate set to 0.59 [kg/s]
 **   ~~~   ** Controller actuator temperature = 12.20 [C]
 **   ~~~   **   Note: Chilled water coils should be reverse action and the entering chilled

 SetpointManager:OutdoorAirReset,
  Chilled Water Loop ChW Temp Manager,  !- Name
  Temperature,             !- Control Variable
  12.2,                    !- Setpoint at Outdoor Low Temperature {C}
  15.6,                    !- Outdoor Low Temperature {C}
  6.7,                     !- Setpoint at Outdoor High Temperature {C}
  26.7,                    !- Outdoor High Temperature {C}
  Chilled Water Loop ChW Supply Outlet;  !- Setpoint Node or NodeList Name
```

OK, this is a result of CW temp setpoint control:

![doatofansupply](https://cloud.githubusercontent.com/assets/5778156/18412586/20d498a8-7760-11e6-8375-341491f6fb5c.png)

**Comment by rraustad** [comment]
See PR https://github.com/NREL/EnergyPlus/pull/5855

**Comment by Myoldmopar** [comment]
Closed via #5855
